tools/eslint-rules/required-modules.js

```diff
@@ -79,7 +76,7 @@ module.exports = function(context) {
           }
         );
         missingModules.forEach(function(moduleName) {
-          context.report(
+          report(
```

Comment:
something here fails the linter:
```js
Running JS linter...
./node tools/jslint.js -j 2 -f tap -o test-eslint.tap \
	benchmark doc lib test tools
/usr/home/iojs/build/workspace/node-test-linter/tools/eslint/lib/rule-context.js:210
        this.eslint.report(
            ^

TypeError: Cannot read property 'eslint' of undefined
    at report (/usr/home/iojs/build/workspace/node-test-linter/tools/eslint/lib/rule-context.js:210:13)
    at /usr/home/iojs/build/workspace/node-test-linter/tools/eslint-rules/required-modules.js:79:11
    at Array.forEach (native)
    at Linter.Program:exit (/usr/home/iojs/build/workspace/node-test-linter/tools/eslint-rules/required-modules.js:78:24)
    at emitOne (events.js:101:20)
    at Linter.emit (events.js:188:7)
    at NodeEventGenerator.applySelector (/usr/home/iojs/build/workspace/node-test-linter/tools/eslint/lib/util/node-event-generator.js:265:26)
    at NodeEventGenerator.applySelectors (/usr/home/iojs/build/workspace/node-test-linter/tools/eslint/lib/util/node-event-generator.js:294:22)
    at NodeEventGenerator.leaveNode (/usr/home/iojs/build/workspace/node-test-linter/tools/eslint/lib/util/node-event-generator.js:317:14)
    at CodePathAnalyzer.leaveNode (/usr/home/iojs/build/workspace/node-test-linter/tools/eslint/lib/code-path-analysis/code-path-analyzer.js:622:23)
    at Traverser.leave (/usr/home/iojs/build/workspace/node-test-linter/tools/eslint/lib/linter.js:925:36)
gmake: *** [Makefile:876: jslint-ci] Error 2
/usr/home/iojs/build/workspace/node-test-linter/tools/eslint/lib/rule-context.js:210
        this.eslint.report(
            ^
```
https://ci.nodejs.org/job/node-test-linter/10203/console